### PR TITLE
Division of AMPL code in functional blocks

### PR DESCRIPTION
**Pull request by p-arvy**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

Yes, #20.

**What kind of change does this PR introduce?**

The AMPL code of OpenReac is divided into functional blocks.

**What is the current behavior?**

These blocks are grouped into `reactiveopf.mod` and `reactiveopf.run` files.

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: open-reac/README.md [hunk 1]
```diff
@@ -57,8 +57,11 @@ each serving a specific function:
 - `reactiveopf.dat` defines the network data files imported (files with
   *ampl_* prefix), and the files used to configure the run (files with *param_* prefix).
   Refer to section [3](#3-input).
-- `reactiveopf.mod` defines the sets, parameters and optimization problems (CC, DCOPF, ACOPF) solved in `reactiveopf.run`. 
-  Refer to sections [5](#5-slack-bus--main-connex-component), [6](#6-direct-current-optimal-power-flow) and [7](#7-alternative-current-optimal-power-flow).
+- `iidm_importer.mod`, `or_param_importer.mod` and `commons.mod` defines the sets and parameters of the optimization.
```
Comment (So-Fras) [inline open-reac/README.md:60]:
```suggestion
- `iidm_importer.mod`, `or_param_importer.mod` and `commons.mod` define the sets and parameters of the optimization.
```

File: open-reac/README.md [hunk 2]
```diff
@@ -57,8 +57,11 @@ each serving a specific function:
...
+- `connected_component.mod`, `dcopf.mod` and `acopf.mod` defines the optimization problems solved in `reactiveopf.run`.
```
Comment (So-Fras) [inline open-reac/README.md:61]:
```suggestion
- `connected_component.mod`, `dcopf.mod` and `acopf.mod` define the optimization problems solved in `reactiveopf.run`. 
```

**Review by klorel — APPROVED** [review]

File: open-reac/src/main/resources/openreac/acopf.mod [hunk 3]
```diff
@@ -0,0 +1,250 @@
+###############################################################################
+#
+# Copyright (c) 2022 2023 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
```
Comment (So-Fras) [inline open-reac/src/main/resources/openreac/acopf.mod:7]:
```suggestion
# SPDX-License-Identifier: MPL-2.0
```
Could you add the license identifier on .mod, .run and .dat files?

**Review by So-Fras — CHANGES_REQUESTED** [review]
Only minor comments!

File: open-reac/README.md (hunk 2 above)
Comment (p-arvy) [reply open-reac/README.md]: Thank you for the proof-reading @So-Fras. Modifications applied :)

File: open-reac/src/main/resources/openreac/acopf.mod (hunk 3 above)
Comment (p-arvy) [reply open-reac/src/main/resources/openreac/acopf.mod:8]: Done. Ty :)

**Review by So-Fras — APPROVED** [review]

**Pull request merged**